coders/art.c

```diff
@@ -325,6 +325,8 @@ static MagickBooleanType WriteARTImage(const ImageInfo *image_info,Image *image,
   (void) WriteBlobLSBShort(image,0);
   (void) WriteBlobLSBShort(image,(unsigned short) image->rows);
   quantum_info=AcquireQuantumInfo(image_info,image);
+  if(quantum_info==NULL)
```

Comment:
Could you change this to: `if (quantum_info == (QuantumInfo *) NULL)`? And also include the type for all the other checks?

Reply:
Ok, i will change all of them